### PR TITLE
Critical issue on the release version

### LINKED ISSUE #799: Critical issue on the release version
Hi, I've performed the UAT and there are some critical points that need to be fixed before it released because will end up disturbing the field data collection process:
- Please kindly remove the Medical record ID as required. I think I have addressed this several times, it has been fixed, right?
- All text-related answer fields in symptom and history should allow spaces. Please kindly fix that.
- Please remove the onset date on the Sign and Symptom form as required. Later we will enhance the question, but for now, just don't make it required.
- Please show the post-partum and pregnant question in the release version
- The test taker/name of sampler (Nama Pengambil Sampel) in sample collection form should not be automatically entered (The issue has been solved, right?), and there is one additional column for test taker ID below that question (ID Pengambil Sampel)
- Please kindly move the doctor's name and doctor's phone number to symptom and history form, with additional Doctor's ID/ID Petugas Pemeriksa and Phone Number question.

There are also some enhancement features that need to be solved, which will be opened in separate issues, but the critical points above should be addressed in this release. The documentation of the UAT will be sent by email.

@mwkhan1983 @syedowaisali90 @vincent-karuri

### PR DESCRIPTION
**Pull request by syedowaisali90**
close #799

### CODE REVIEW

File: rdt-open-reader/src/covidIndonesia/assets/json.form/patient-diagnostics-form-original.json [hunk 1]
```diff
@@ -114,6 +153,52 @@
...
+        },
+        "v_max_length": {
+          "value": "13",
+          "err": "Phone number cannot be more than 13 digits"
+        },
+        "v_min_length": {
+          "value": "8",
+          "err": "Phone number cannot be less than 8 digits"
+        }
+      },
```
Comment (vincent-karuri) [inline rdt-open-reader/src/covidIndonesia/assets/json.form/patient-diagnostics-form-original.json:201]: I don't think we need this. We should just have the healthworker details.

File: rdt-open-reader/src/covidIndonesia/assets/json.form/patient-diagnostics-form-original.json [hunk 2]
```diff
@@ -509,11 +594,7 @@
         "hint": "Date of onset of symptoms",
         "expanded": false,
         "min_date": "today-100y",
-        "max_date": "today",
-        "v_required": {
-          "value": "true",
-          "err": "This field is required"
-        }
+        "max_date": "today"
```
Comment (vincent-karuri) [inline rdt-open-reader/src/covidIndonesia/assets/json.form/patient-diagnostics-form-original.json:597]: Could we add the sampler ID and phone number https://github.com/onaio/rdt-standard/issues/799#issuecomment-784321116?

Comment (syedowaisali90) [reply rdt-open-reader/src/covidIndonesia/assets/json.form/patient-diagnostics-form-original.json:597]: I have added a sampler id and phone number on sample-collection-form https://github.com/onaio/rdt-standard/blob/440b5fb4e1e0cc4ba215a600e727728f32dc4051/rdt-open-reader/src/covidIndonesia/assets/json.form/sample-collection-form-original.json#L73-L99

Comment (vincent-karuri) [reply rdt-open-reader/src/covidIndonesia/assets/json.form/patient-diagnostics-form-original.json:597]: cool

File: rdt-open-reader/src/covidIndonesia/assets/json.form/patient-diagnostics-form-original.json (hunk 1 above)
Comment (syedowaisali90) [reply rdt-open-reader/src/covidIndonesia/assets/json.form/patient-diagnostics-form-original.json]: fields removed.

**Review by vincent-karuri — APPROVED** [review]

**Pull request merged**